orgSeries-taxonomy.php

```diff
@@ -119,7 +119,7 @@ function set_series_order($postid = 0, $series_part = 0, $series_id, $is_publish
  	$series_posts = get_series_order($post_ids_in_series, $postid, $series_id, true, false);
 	$parts_array = array();
 
-	$total_posts = count( $series_posts ) + 1;
+	$total_posts = is_array($series_posts) ? count( $series_posts ) + 1 : 1;

```

Comment:
This seems odd that we're adding a count here.  But I'm preserving existing behaviour (hence the default of 1 when not an array).